### DISCUSSION TITLE
I just want to start a service that uses the openai big model interface, what should I do?

### DISCUSSION THREAD

**Discussion by ye7love7** [TabbyML/tabby#2943] [General]
quick start only Provides commands to start large models at the same time, I want to simply start a service and use the openai interface provided by vllm or ollama, which is the more common way to do it, the model platform and the service application should be separated.please replenishment a programme,thanks a lot.

**Comment by wsxiaoys** [comment]
Tabby supports connect to a model backend through API - see https://tabby.tabbyml.com/docs/administration/model/ for configuration examples
